packages/wallet/src/persistence/pouchdbStores/pouchdbSupplyDistributionStores.ts

```diff
@@ -0,0 +1,33 @@
+import { CreatePouchdbStoresDependencies } from './types';
+import { EMPTY, map } from 'rxjs';
+import { PouchdbDocumentStore } from './PouchdbDocumentStore';
+import { StakeSummary, SupplySummary } from '@cardano-sdk/core';
+import { SupplyDistributionStores } from '../types';
+
+export class PouchdbStakeSummaryStore extends PouchdbDocumentStore<StakeSummary> {}
+export class PouchdbSupplySummaryStore extends PouchdbDocumentStore<SupplySummary> {}
+
+/**
+ * @param {string} baseName used to derive underlying db names, like a network ID and/or wallet name
+ */
+export const createPouchdbSupplyDistributionStores = (
```

Comment:
```suggestion
export const createPouchDbSupplyDistributionStores = (
```
Although, the existing names all use the same capitalisation in _master_, so this would be a separate refactor commit to change all. Can defer if preferred.

Reply:
79f93a8c7b79f43a2aedb0edb505dcf39d30cf49